staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -91,7 +93,8 @@ func (r *proxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	if !handlingInfo.serviceAvailable {
+	if !handlingInfo.local && !handlingInfo.serviceAvailable {
+		unavailableRequestCounter.WithLabelValues(handlingInfo.name)
```

Comment:
aren't these missing an `.Inc()` on the end to actually record something?

Reply:
handlingInfo.local is handled above... shouldn't be necessary to check again

Reply:
oh i just saw this, please disregard my other comment.

Reply:
fixed